### PR TITLE
v4.2 Prep Work

### PR DESCRIPTION
**Pull request by kingthorin**
- Create `.github/www/v42`
- Update `.github/www/stable/prepend.txt` to include the new breadcrumb component.
- Update credits and history.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
I'll rebase this PR after: https://github.com/OWASP/wstg/pull/612is merged.

File: document/2-Introduction/README.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ Denis Verdon, Head of Information Security at [Fidelity National Financial](http
...
 
 If identifiers are used without including the `<version>` element then they should be assumed to refer to the latest Web Security Testing Guide content. Obviously as the guide grows and changes this becomes problematic, which is why writers or developers should include the version element.
 
 #### Linking
 
-Linking to Web Security Testing Guide scenarios should be done using versioned links not `stable` or `latest` which will definitely change with time. However, it is the project team's intention that versioned links not change. For example: `https://owasp.org/www-project-web-security-testing-guide/v41/4-Web_Application_Security_Testing/01-Information_Gathering/02-Fingerprint_Web_Server.html`. Note: the `v41` element refers to version 4.1.
+Linking to Web Security Testing Guide scenarios should be done using versioned links not `stable` or `latest` which will definitely change with time. However, it is the project team's intention that versioned links not change. For example: `https://owasp.org/www-project-web-security-testing-guide/v42/4-Web_Application_Security_Testing/01-Information_Gathering/02-Fingerprint_Web_Server.html`. Note: the `v41` element refers to version 4.2.
```
Comment (ThunderSon) [inline document/2-Introduction/README.md:76]:
```suggestion
Linking to Web Security Testing Guide scenarios should be done using versioned links not `stable` or `latest` which will definitely change with time. However, it is the project team's intention that versioned links not change. For example: `https://owasp.org/www-project-web-security-testing-guide/v42/4-Web_Application_Security_Testing/01-Information_Gathering/02-Fingerprint_Web_Server.html`. Note: the `v42` element refers to version 4.2.
```

File: document/6-Appendix/E-History.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+- September, 2014: OWASP Testing Guide, Version 4.0
+- September, 2008: OWASP Testing Guide, Version 3.0
+- December, 2006: OWASP Testing Guide, Version 2.0
+- July, 2004: OWASP Web Application Penetration Checklist, Version 1.1
+- December, 2004: The OWASP Testing Guide, Version 1.0
 
 ## Leaders
 
-- Matteo Meucci: OWASP Testing Guide Lead since 2007.
+- Matteo Meucci: OWASP Testing Guide Lead since 2007-2020.
```
Comment (ThunderSon) [inline document/6-Appendix/E-History.md:14]:
```suggestion
- Matteo Meucci: OWASP Testing Guide Lead 2007-2020.
```

**Review by ThunderSon — CHANGES_REQUESTED** [review]

**Comment by ThunderSon** [comment]
@kingthorin not sure you committed the changes? It looks similar.

**Comment by kingthorin** [comment]
It is the same it’s just rebased to pickup the change on the failing link.

**Comment by ThunderSon** [comment]
@kingthorin you set the conversations are resolved without having any changes in. Is that what you wanted?

**Comment by kingthorin** [comment]
Yup I had accidentally overwritten their commits, fixed now.

File: document/1-Frontispiece/README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 OWASP thanks the many authors, reviewers, and editors for their hard work in bringing this guide to where it is today. If you have any comments or suggestions on the Testing Guide, please feel free to open an Issue or submit a fix/contribution via Pull Request to our [GitHub repository](https://github.com/OWASP/wstg/).
 
-## Version 4.1
+## Version 4.2
```
Comment (ThunderSon) [inline document/1-Frontispiece/README.md:13]: I think before merging, we need to change the description of this part.
Let's review the change logs (when you can generate them, no rush), and let's set it accordingly.

Comment (kingthorin) [reply document/1-Frontispiece/README.md:13]: Yup that does make sense, I did  have a quick look at it when I was putting this PR together but I couldn't think of what I'd rather say so I just forged ahead anyway.

I think doing it after prepping change notes and repo history is a good move.

Comment (kingthorin) [reply document/1-Frontispiece/README.md:13]: Let's tackle that as along PR though. It'll probably take me a few days (if not longer) to tackle the notes, then swing back around to this.

**Review by ThunderSon — APPROVED** [review]

**Pull request merged**